src/test/resources/net/pms/util/prettified_filenames_metadata.json

```diff
@@ -178,6 +193,7 @@
 		}
 	},
 	{
+		"id": 16,
```

Comment:
Why do these need IDs when the filename is already unique?

Reply:
That's a good thought to anticipate that. I'm not sure we would need a manual unique key in that case either. If we need an integer we can get the items' position in the JSON automatically, or just concatenate two property values to make a unique one (like `context + filename`).
I am just not a fan of manually incrementing numbers :)